README.md

```diff
@@ -47,36 +48,103 @@ Demo: https://img-v3.getdemo.dev/screenshot/BiI0D6ygq3.mp4
 
 
 ##### Step 4: Extend `pageobjects` and variables in `constructor`
-Extend all your `pageobjects` to `const Page = require('./../vaah-webdriverio/Page');`, 
+Extend all your `pageobjects` to `const Page = require('./../Page');`, 
 
-Example: `pageobjects/Login.page.js`: 
+Example: `pageobjects/signup.page.js`: 
 ```js
-const Page = require('./../vaah-webdriverio/Page');
-
-class Login extends Page {
+const Page = require('./../Page');
+const Sl = require('./../Selector');
+class SignupPage extends Page {
 
     constructor() {
         super();
-        this.page.id = "LI"; // Page ID, Please keep this unique for all the pages.
-        this.page.name = "Login";
-        this.page.path = "login";
-        this.page.url = this.base_url + this.page.path;
+        this.params.page.id = "SU";
+        this.params.page.name = "Sign-Up";
+        this.params.page.url = this.signup_url;
     }
-
+    
     open (url=null) {
         if(url)
         {
             this.page.url = url
         }
         return super.open(this.page.url);
     }
-    
+
+    async signUpButtonText(data,assert)
+    {
+        await expect(Sl.$(data.selectors.signUp)).toHaveTextContaining(assert);
+    }
+
+    async signUpWithValidData(first_name,last_name,email,password,data,assert)
+    {
+        await this.fillAndSignUp(first_name,last_name,email,password,data)
+        await expect(Sl.$(data.selectors.page_heading)).toHaveTextContaining(assert);
+    }
+    ...
 }
-module.exports = new Login();
+module.exports = new SignupPage();
 ```
-Demo: https://img-v3.getdemo.dev/screenshot/iTSi72u1p3.mp4
+Demo: https://img-v4.getdemo.dev/screenshot/phpstorm64_EZkTiTFfTr.mp4
 
-#### step 5: All the methods present in Assert class under Assets in vaah-webdriverio should be prefixed with "async" to run the methods in async mode in the test scripts.
+#### step 5: All the values which are required for the `signup.page.js` and `signup.spec.js` should be present in `signup.js` under `data` and for the values which are required for the test scripts, it should be arranged in `groups` and `tests` in JSON format.
+Example:
+```js
+class Signup{
+
+    constructor() {
+        this.selector_type = 'placeholder';
+
+        this.params = {
+            group:{
+                count: null,
+                name: null,
+            }
+        }
+
+        this.selectors = {
+            first_name: 'First Name',
```

Comment:
@satyajeet-s001 how selector will be a `First Name`, it will be a `data-testid` attribute, explain that this is a `data-testid` attribute value